CONTRIBUTING.md

```diff
@@ -49,3 +49,12 @@ branches.
 
 Please see our [git style guide](https://github.com/alphagov/styleguides/blob/master/git.md)
 which describes how we prefer git history and commit messages to read.
+
+### Review apps
+
+When a pull request is opened Heroku may create a [review app](https://devcenter.heroku.com/articles/github-integration-review-apps#viewing-review-apps)
+that will allow you and reviewers to preview how your changes will appear for users.
+If a review app is not created automatically you can ask someone from the
+Prototype Kit team to create one.
```

Comment:
Is there a PK team yet or it just us at the Design System? If the latter, we could always say something like:

> If a review app is not created automatically, please [contact the Design System team](https://design-system.service.gov.uk/get-in-touch/) and we'll create one for you.

Reply:
I _think_ we probably want users to ask us when they create the pull request (at least to start with) rather than contacting us via email or Slack, so we might want to be careful about signposting that option too heavily.